codemirror/codemirror.d.ts

```diff
@@ -786,8 +786,8 @@ declare module CodeMirror {
         and thus the browser's text search works on it. This will have bad effects on performance of big documents. */
         viewportMargin?: number;
 
-        /** Optional lint configuration to be used in conjunction with CodeMirror's linter addon. */
-        lint?: LintOptions;
+        /** LintOptions|boolean. Optional lint configuration to be used in conjunction with CodeMirror's linter addon. */
+        lint?: any;

```

Comment:
How about declaring it as `LintOptions|boolean` ?
The current released version of TypeScript supports that syntax.